src/pages/profile/[id].tsx

```diff
@@ -0,0 +1,15 @@
+import { type NextPage } from "next";
+import {CreateBlogPostWidget} from "~/components/create-blog-post-widget";
+import { Layout } from "~/components/Layout";
+import {ProfileTabbedView} from "~/components/ProfileTabbedView";
+
+const Profile: NextPage = () => {
```

Comment:
I would recommend to parse the URL parameters here and pass it down to the components, that way, it is cleaner inside the components and components are not doing something that is page component's responsibility

Reply:
done